festim/subdomain/volume_subdomain.py

```diff
@@ -37,9 +37,20 @@ def locate_subdomain_entities(self, mesh, vdim):
         Returns:
             entities (np.array): the entities of the subdomain
         """
+        self.check_borders_within_domain(mesh)
+
         entities = locate_entities(
-            mesh,
+            mesh.mesh,
```

Comment:
We shouldn't to do the check here, so no need to change the mesh argument (note: if you wanted to change it, you shoud've changed the docstrings too)